apps/events/tasks.py

```diff
@@ -53,9 +53,9 @@ def fire_trigger(trigger_id, session_id):
 
 
 def _get_messages_to_fire():
-    return ScheduledMessage.objects.filter(is_complete=False, next_trigger_date__lte=functions.Now()).select_related(
-        "action"
-    )
+    return ScheduledMessage.objects.filter(
+        is_complete=False, action__isnull=False, next_trigger_date__lte=functions.Now()
```

Comment:
will you remove this filter once the rest is implemented?

Reply:
yes that was the idea but I was hoping to merge this prior to the other being merged just for ease